tests/services/test_rdm_service.py

```diff
@@ -9,21 +9,37 @@
 
 """Service level tests for Invenio RDM Records."""
 
+from collections import namedtuple
+
 import pytest
 from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_vocabularies.proxies import current_service as vocabulary_service
 from marshmallow import ValidationError
 
 from invenio_rdm_records.proxies import current_rdm_records
 
+RunningApp = namedtuple("RunningApp", [
+    "app", "location", "superuser_identity", "resource_type_item"
+])
+
+
+@pytest.fixture
+def running_app(app, location, superuser_identity, resource_type_item):
```

Comment:
it feels a bit duplicated... If we want to make if "specific" for tests at this level... shall we add `es_clear` and maybe even `minimal_record` (yeah, the latter does not really much with the name).